libr/asm/arch/8051/8051_ass.c

```diff
@@ -442,29 +442,14 @@ static bool parse_register(char const* register_input, ut8* hex_out) {
 	return true;
 }
 
-/**
- * attempts to parse the given string as an 8bit-wide address
- */
+  /*
+  * attempts to parse the given string as an 8bit-wide address
```

Comment:
can you rename the function name, because it have nothing to do with what the comment describes. thiçs way the comment can be removed

Reply:
@trufae Do you think I modified this function too much? I did not want to change it's behavior.

Reply:
If it needs to bw done it must be done its never too much (: